pkgs/misc/emulators/ryujinx/updater.sh

```diff
@@ -1,10 +1,33 @@
-#!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl jq common-updater-scripts dotnet-sdk_5
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p common-updater-scripts nix-prefetch-git jq dotnet-sdk_5
```

Comment:
grep and gnused are missing.

Reply:
Done :+1:

Reply:
You are missing gnused.

Reply:
Not sure how i forgot about that, but it should be in there now :)